### DISCUSSION TITLE
block_list for allowing multiple domains

### DISCUSSION THREAD

**Discussion by petsneakers** [mitmproxy/mitmproxy#6196] [General]
Using `block_list` to drop everything except a single domain looks like
```
:!~d ^foo\.com$:444
```

Is there a way to do this for multiple domains?  I'd have expected this to work but it allows all traffic:
```
:!~d ^(foo\.com|bar\.com)$:444
```

**Comment by mhils** [comment]
I don't see how this regex would ever match with the $ in the middle?

**Comment by petsneakers** [comment]
The regex is just part of the filter:

- `^foo\.com$`
- `^(foo\.com|bar\.com)$`
